utils/db_benchmark/benchmark_streamlit.py

```diff
@@ -0,0 +1,413 @@
+"""
+Benchmark Streamlit
+-------------------
+This module provides a streamlit interface to view benchmark results.
+The interface has 4 tabs:
+
+- "Benchmark sets" -- Here you can add or remove benchmark set files (generated by `save_results_to_file` function).
+- "View" -- Here you can view various stats for each benchmark in the set (as well as add benchmarks to compare tab).
+- "Compare" -- Here you can compare benchmarks added via "View" tab.
+- "Mass compare" -- This tab lets you compare all benchmarks in a particular set.
```

Comment:
I think we should really add some descriptions on how to *use* these tabs (and the whole page) - onto the page itself.